sdk/lib/atl/atlbase.h

```diff
@@ -766,7 +766,11 @@ class CAtlExeModuleT : public CAtlModuleT<T>
 
     static HRESULT InitializeCom()
     {
+#if defined(_ATL_FREE_THREADED)
         return ::CoInitializeEx(NULL, COINIT_MULTITHREADED);
+#else
+        return ::CoInitialize(NULL);
+#endif
```

Comment:
What about just keeping the original CoInitializeEx call but just using `COINIT_APARTMENTTHREADED` instead as the flag? Something like
```suggestion
#ifdef _ATL_FREE_THREADED
        static const COINIT coInit = COINIT_MULTITHREADED;
#else
        static const COINIT coInit = COINIT_APARTMENTTHREADED;
#endif
        return ::CoInitializeEx(NULL, coInit);
```